spec/models/manageiq/providers/vmware/cloud_manager/refresher_spec.rb

```diff
@@ -58,7 +58,7 @@
   end
 
   def assert_table_counts
-    expect(ExtManagementSystem.count).to eq(1)
+    expect(ExtManagementSystem.count).to eq(2)  # cloud_manager + network_manager
```

Comment:
for the other provider refresh spec, I do test both refreshes here. So we can test also the connection to Vms

on L46, just call also EmsRefresh.refresh(@ems.network_manager)